hazelcast-hibernate5/src/main/java/com/hazelcast/hibernate/CacheEnvironment.java

```diff
@@ -58,6 +58,7 @@
     /**
      * Property to configure Hazelcast client group password
      */
+    @Deprecated
```

Comment:
We are removing password related stuff so why are we keeping that?

Reply:
removed completely in 65bd7ca240115334551b8c44392d05317da2cf6d